spec/features/user_writes_review_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+feature "User writes a review for a Spot" do
+
+# As a user
+# I want to write a review for a study spot
+# to help other users decide where to study.
+
+# Acceptance Criteria
+# - User can add a review
+
+  scenario 'I want to review a study spot' do
+    visit '/spots/1/reviews/new'
+    fill_in 'rating', with: '4'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.